packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/configuration.md

```diff
@@ -1,8 +1,33 @@
+## Verify that the Interaction Code grant type is enabled
+
+<ApiLifecycle access="ea" />
+
+The Interaction Code grant type is a sign-in flow used by embedded applications to connect to Okta. The **Embedded widget sign-in support** setting controls this grant type for all OpenID Connect and API access management servers in your org. With this setting enabled, you can allow or deny apps the ability to use embedded sign-in flows across your entire org. See [Interaction Code grant type](/docs/concepts/interaction-code/) for detailed information on this grant type and how to use it.
+
+> **Note:** Super Admin permissions are required. If you don’t have those permissions for your org, contact your administrator.
+
+1. To access this setting, from your Okta org's [Admin Console](/docs/concepts/okta-organizations/#admin-console), go to **Settings** > **Account** > **Embedded widget sign-in support**.
+1. Click **Edit**.
+1. If the checkbox isn’t selected, select it if you want to use the Interaction Code as a grant type for your OpenID Connect app integrations and authorization servers.
```

Comment:
I wonder if you need the conditional phrase, especially given the sentence after this one.
```suggestion
1. Select the checkbox if you want to use the Interaction Code as a grant type for your OpenID Connect app integrations and authorization servers.
```